unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/PlacesAndEventsSectionComponentView.cs

```diff
@@ -70,16 +79,21 @@ public void SetActive(bool isActive)
     {
         canvas.enabled = isActive;
 
+        //Temporary untill the full feature is released
+        subSectionSelector.GetSection(FAVORITES_SUB_SECTION_INDEX)?.GetGameObject().SetActive(DataStore.i.HUDs.enableFavoritePlaces.Get());
```

Comment:
Although this is temporary, I still don't like it, it feels very forced.

We really need the subSectionSelector to support having a disabled subsection based on a feature flag.

Example:

```C#
if (!DataStore.i.HUDs.enableFavoritePlaces.Get())
    subSectionSelector.DisableSection(FAVORITES_SUB_SECTION_INDEX);
```

Reply:
Alright will work on it :)